c2corg_ui/static/img/logo2.svg

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I just wonder if we could have used the standard one and apply some CSS on it to change the color (orange to white)? Else they are identical. I don't have the answer. It's not a disaster if not but we could then save the extra 19 KB.